### DISCUSSION TITLE
Posibility of enhancement to html logger to add option of navigating to host?

### DISCUSSION THREAD

**Discussion by haderp** [jamesoff/simplemonitor#1119] [Q&A]
Would an enhancement to the html logger output to introduce the ability to click the name and have it navigate to the hostname/ip be a possible in the future?  I know it's possible, but was unsure how hard it would be to add a field to a monitor that enabled/disabled browsing to the hostname/ip:port and subsequently updates the template to make the hostname a link.  

Kind Regards

**Comment by jamesoff** [comment]
Yes, that should be pretty straightforward to do; it sounds like it would be similar to the [failure_doc](https://simplemonitor.readthedocs.io/en/latest/monitors.html#confval-failure_doc) option except you want it to show up in the HTML status page, kind of thing?

Edit: I created https://github.com/jamesoff/simplemonitor/issues/1120 to track this :)

**Comment by haderp** [comment]
Yes, that is similar to what I was thinking.  In my mind I envisioned that an additional configuration variable would exist for the individual monitor setup called "browseable=true/false" default false and if the variable is set and true another variable called "browseable_url=x.x.x.x" would be added.  Then in the HTML generation if the "browseable=true" and "browseable_url=x.x.x.x" in the generation of the page it would flag the option as set and instead of inserting monitor name it would wrap the monitor name in an HTML a href that when clicked opens a new tab/browser to the set ip/url leaving the monitor up.  That is only what I envisioned however, and I know that what seems simple is not always the case and so I can only leave those intricacies up to the person that ultimately is being asked to do it.  I liked the approach since it does not add clutter to the HTML page, but makes browsing to the host easy to do.  Thanks for adding it as an enhancement, any option that makes it possible to click and go would be greatly appreciated since many of my hosts have a web ui, but not necessarily on a standard port.  

Thank You 

Edit: A word

**Comment by jamesoff** [comment]
Sounds good to me; I'd probably just have the `browsable_url` (or similar) setting; if it's present then the link appears, else it doesn't. Saves needing to separately "enable" it :)

**Reply by haderp** [reply]
That is a much more practical and simple implementation, I like it.  Thanks for taking the time to entertain my request.  After testing a lot of monitoring solutions yours is by far the best I have used that isn't overkill for a small environment or resource hungry.  Thanks for your work it is appreciated.  Cheers
